packages/flutter/lib/src/widgets/page_view.dart

```diff
@@ -383,11 +383,13 @@ class _PagePosition extends ScrollPositionWithSingleContext implements PageMetri
 
   @override
   double? get page {
+    // Get the page value before the content dimension is determined, do not throw an assertion.
```

Comment:
Can you remove these comments? The content dimensions are not the only thing being evaluated here.